dashboard/package.json

```diff
@@ -4,9 +4,13 @@
   "description": "",
   "main": "index.js",
   "dependencies": {
+    "@emotion/react": "^11.10.6",
+    "@emotion/styled": "^11.10.6",
+    "@mui/icons-material": "^5.11.11",
+    "@mui/material": "^5.11.13",
```

Comment:
do we need this ? cant we just use icons ?

Reply:
@chad11111 i'll try to check with icons i am not sure about that. i am comfortable with material-ui and its a  bit more styled but if you want me to do with icons i'll look into it.

Reply:
We dont want to use mui.
Please only use the icons.